<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBoxImporto">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSaldo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Saldo attuale:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeposita">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposita</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPreleva">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ritira</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salva conto </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Carica conto esistente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBonifico">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Invia bonifico</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDestinatario">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nome Destinatario</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listTransactions">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSaldo_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>I tuoi movimenti</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSaldo_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserisci importo poi scegli l'operazione</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
